diff --git a/node_modules/uniforms/src/types.ts b/node_modules/uniforms/src/types.ts
index e209f8c..0fbcfc9 100644
--- a/node_modules/uniforms/src/types.ts
+++ b/node_modules/uniforms/src/types.ts
@@ -81,7 +81,7 @@ export type ValidateMode = 'onChange' | 'onChangeAfterSubmit' | 'onSubmit';
 
 // Explicit declaration to allow users extend it directly.
 // Would it be possible not to use `declare` here?
-declare module '.' {
+declare module 'uniforms' {
   interface FilterDOMProps {
     allowedValues: never;
     changed: never;
